CURATORS.md

```diff
@@ -294,6 +294,8 @@ problems on nightly or LTS major, you need to fix build-constraints.yaml (see
 info above). 
 
 ### Building LTS minor releases
+Before running the build, please make sure that the Dockerfile in `automated/dockerfiles/lts-X.Y` is up to date, where X is the major version that you're building and Y is the latest minor version of X. If any changes need to be made, (eg, new GHC version), copy `lts-X.Y/Dockerfile` to `lts-X.Z/Dockerfile`, where Z is the minor version you're building, and include the new changes.
```

Comment:
This is slightly incorrect in that `automated/dockerfiles/lts-X.Y` may not exist since, for example, the lts-15.0 Dockerfile will be reused for lts-15.1, 15.2, etc. if there's no "overrriding" Dockerfile.  So it's whatever the most recent Dockerfile for `lts-x.*` that should be copied.

Reply:
Yes I did mention `X is the major version that you're building and Y is the latest minor version of X`, but I guess that isn't very obvious to the reader, how may I improve this?

Reply:
I think just replacing

> and Y is the latest minor version of X

with

> and Y is the latest minor version of X for which a Dockerfile exists

will do the trick.  I also find this tricky to word correctly.